webpack.config.js

```diff
@@ -248,13 +248,13 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 					test: /\.(sc|sa|c)ss$/,
 					use: [
 						MiniCssExtractPluginWithRTL.loader,
-						'css-loader',
 						{
-							loader: 'postcss-loader',
+							loader: 'css-loader',
 							options: {
-								plugins: [ require( 'autoprefixer' ) ],
+								importLoaders: 2,
```

Comment:
Is the `importLoaders` option needed for anything? I tried to build CSS both with and without it and the resulting CSS assets are exactly the same. I'm very confused about what exactly all these CSS-ish loaders do. `css-loader` docs say it resolves `@import` statements, but doesn't the `sass-loader` do that for us?

Reply:
I see that [create-react-app does the same thing](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L471).